CONTRIBUTING.md

```diff
@@ -12,8 +12,8 @@ I care about code quality, readability and tests, so please follow the current s
 
 ### Required
 
-* golang, version >= 1.13
-* docker, version >= 19
+* Go, version >= 1.16
```

Comment:
We currently don't have any tracking of version upgrade for cogito. Do you think it makes sense to think of one ? I don't have in mind but might be needed.

Reply:
I think it is OK as-is for the moment.
I plan to move the CI from Travis to GitHub Actions; when that is done, we can reconsider.